src/apps/gie.cpp

```diff
@@ -753,7 +753,7 @@ static PJ_COORD parse_coord(const char *args) {
         // test failures when testing points at edge of grids.
         // For example 1501000.0 becomes 1501000.000000000233
         double d = proj_strtod(prev, &endp);
-        if (*endp != '\0' && !isspace(*endp)) {
+        if (!isnan(d) && *endp != '\0' && !isspace(*endp)) {
```

Comment:
This was the simplest solution here to just say "skip this if statement if we have a NaN" but I'm not sure it is the right decision.